src/main/scala/io/iohk/ethereum/jsonrpc/PersonalService.scala

```diff
@@ -59,6 +59,7 @@ object PersonalService {
   val InvalidAddress = InvalidParams("Invalid address, expected 20 bytes (40 hex digits)")
   val InvalidPassphrase = LogicError("Could not decrypt key with given passphrase")
   val KeyNotFound = LogicError("No key found for the given address")
+  val PassPhraseTooShort = LogicError("Provided passphrase must have at least 8 characters")
```

Comment:
isn't it 7?

Reply:
ps: this can be configured, it's not hardcoded at a certain number, right?

Reply:
wooh embarrassing mistake, fixed in next commit